src/assets/styles/10-settings/_breakpoints.scss

```diff
@@ -0,0 +1,14 @@
+// ----------------------------------
+// UI Settings
+// Breakpoints
+// ----------------------------------
+// Design System 1.0 Alpha
+// 2019-11-11
+// ----------------------------------
+
+$breakpoints: (
```

Comment:
👍 good to see `sm`, `md` etc as opposed to `mobile`, `tablet` etc

Reply:
Agreed - think it's always best to shorthand when it's widely accepted syntax 👍🏻

Reply:
It also ties in with the work Vlad did on typefaces and spacings 👍